compose-custom-attachments/src/main/java/io/getstream/chat/android/compose/customattachments/activity/ChannelsActivity.kt

```diff
@@ -0,0 +1,30 @@
+package io.getstream.chat.android.compose.customattachments.activity
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import io.getstream.chat.android.compose.customattachments.ui.eventAttachmentFactory
+import io.getstream.chat.android.compose.ui.attachments.StreamAttachmentFactories
+import io.getstream.chat.android.compose.ui.channels.ChannelsScreen
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+
+class ChannelsActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val customFactories = listOf(eventAttachmentFactory)
+        val defaultFactories = StreamAttachmentFactories.defaultFactories()
+
+        setContent {
+            ChatTheme(attachmentFactories = customFactories + defaultFactories) {
```

Comment:
The newly created custom attachment factory is also necessary on the channel list screen to show message previews.